odd_collector/adapters/fivetran/domain/connector.py

```diff
@@ -0,0 +1,21 @@
+from dataclasses import dataclass
+from typing import Any
+
+
+@dataclass
+class ConnectorMetadata:
+    id: Any
```

Comment:
Does Fivetran provide information about their types? If yes use them instead of `Any`